.circleci/config.yml

```diff
@@ -6,8 +6,7 @@ version: 2
 jobs:
   build:
     docker:
-      # specify the version you desire here
-      - image: circleci/node:10.15
+      - image: cimg/node:16.14.2
```

Comment:
The warning from linter would also need to be resolved before upgrading to new version.
`warning  'url.parse' was deprecated since v11.0.0. Use 'url.URL' constructor instead`

Reply:
fixed, thanks. btw how do you get the warning?

Reply:
`npm run lint`